operator/Makefile

```diff
@@ -174,6 +174,9 @@ informer-gen: ## Download informer-gen
 lister-gen: ## Download lister-gen
 	GOBIN=$(LOCALBIN) go install k8s.io/code-generator/cmd/lister-gen@v0.23.4
 
+.PHONY: resolve
```

Comment:
what's the reason to introduce this?

Reply:
It's just a shortcut for us instead of `go mod tidy` 😄